CP1404_an/Practical_06/guitars.py

```diff
@@ -0,0 +1,42 @@
+"""
+Prac 06 CP1404 - Guitar class operations :test file
+Expected time: 20min
+Actual time: 30min
+"""
+
+from Practical_06.guitar import Guitar
+
+TITLE = "My guitars!"
+guitars = []
+
+
+def main():
+    """Program that gets and processes user guitar info based on Guitar Class"""
+    print(TITLE)
+    guitar_name = input("Name: ")
+    while guitar_name != "":
+        guitar_year = get_valid_number("Year: ")
+        guitar_cost = get_valid_number("Cost: ")
+        guitar = Guitar(guitar_name, guitar_year, guitar_cost)
+        print(f"{guitar} added")
+        guitar_name = input("Name: ")
+        guitars.append(guitar)
+
+    for i, guitar in enumerate(guitars):
+        category = "(Vintage)" if guitar.is_vintage() else ""
+        print(f"{i + 1} {guitar} {category}")
+
+
+def get_valid_number(prompt):
+    while True:
```

Comment:
I think Mrs. Cynthia does not allow us to use while True.

Reply:
Oh, alright I will adjust this